README.md

```diff
@@ -131,11 +133,17 @@ For full list, see [Cookbook](./doc/README.md#1-cookbook--how-to-guide).
 
 ## Getting Started
 
-The basic common usage of Easegress is to quickly set up proxy for the backend servers. We split it into multiple simple steps to illustrate the essential concepts and operations.
+The basic usage of Easegress is to quickly set up a proxy for the backend
+servers. In this section, we will first set up a reverse proxy, and then
+demonstrate the API orchestration feature by including more components in the
+configuration, we will also show the essential concepts and operations of
+Easegress.
 
 ### Setting up Easegress
 
-We can download the latest or history binaries from the [release page](https://github.com/megaease/easegress/releases). The following shell script will do:
+We can download the latest or history binaries from the
+[release page](https://github.com/megaease/easegress/releases). The following
+shell script will:
```

Comment:
```suggestion
shell script will be:
```

Reply:
I don't think we need to add the word `be`, as the full sentences are:
```
The following shell script will download and extract the latest binaries to `./easegress` folder  
```
and
```
The following shell script will install the Easegress Systemd service.
```

Reply:
oops, you're right